src/App.vue

```diff
@@ -237,6 +237,21 @@ export default defineComponent({
 
     window.api.send("getLangPath");
 
+    window.api.receive("ad4mAgentInit", async (isAlreadySignedUp: boolean) => {
+      if (!isAlreadySignedUp) {
+        console.log(
+          "New agent and thus creating a Flux perspective & updating public profile"
+        );
+        const agentPerspective = await ad4mClient.perspective.add(
+          "My flux perspective"
+        );
+        await this.userStore.addFluxPerspectiveId(agentPerspective.uuid);
+        await ad4mClient.agent.updatePublicPerspective({
+          links: [],
+        } as PerspectiveInput);
```

Comment:
We need to be careful about resetting a users profile since they might already have a profile from other ad4m languages. Id remove this for now unless there is a reason you need to be sure the profile is cleared?

Reply:
This will only run when they are creating a new user right so they will no links

Reply:
ah okay I missed that detail, but in that case the public perspective should already be empty right? Or is it empty but not published into the agent-language and this step just ensures the profile is atleast published in the agent-language